landscape/client/deployment.py

```diff
@@ -158,6 +158,12 @@ def make_parser(self):
             help="Repository profiles manage the files in "
             "/etc/apt/sources.list.d",
         )
+        parser.add_option(
+            "--snap-monitor-interval",
+            default=30 * 60,
```

Comment:
Could be 15 minutes. It's cached

Reply:
Actually looking at the output, looks like there are timestamps in there. So best to strip them out so DataWatcher can cache the output and it's not sent every n minutes. Although the KeyDict selects the needed values, the caching should happen before